Makefile

```diff
@@ -51,14 +51,18 @@ OPERATOR_NAME ?= node-maintenance
 # IMAGE_TAG_BASE defines the docker.io namespace and part of the image name for remote images.
 # This variable is used to construct full image tags for bundle and catalog images.
 #
-# For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
```

Comment:
why?  its for a CatalogSource, seems appropriate

Reply:
No it was used, for building the index, but we have used the term `catalog` instead of `index`, I can't recall why, see lines 264 and 269 in the Makefile (before the PR).

Reply:
We didn't come up with the term, whats how the operator-sdk creates the Makefile